pkg/env-tests/scheduler_test.go

```diff
@@ -338,4 +341,128 @@ var _ = Describe("Scheduler", Ordered, func() {
 			Expect(podMap).To(HaveLen(deviceNum), "Expected exactly %d pods to be scheduled", deviceNum)
 		})
 	})
+
+	Context("priority based fair share", func() {
+		var priorityBasedFairShareSchedulerConf = &conf.SchedulerConfiguration{
```

Comment:
Can we outsource this to some kind of `DefaultSchedulerConfiguration`, that you will use and only override `priorityBasedFairShare`?
The reason it's important is that when we make later changes like add/modify plugins, we will not remember to update it in all the places, and each test will check different configurations.
I started working on something like this but got distracted: https://github.com/NVIDIA/KAI-Scheduler/pull/228/files

If it's too much then LMK and I'll create a dedicated ticket